src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1384,7 +1384,7 @@ private IWhileLoopOperation CreateBoundWhileStatementOperation(BoundWhileStateme
             ITypeSymbol type = null;
             Optional<object> constantValue = default(Optional<object>);
             bool isImplicit = boundWhileStatement.WasCompilerGenerated;
-            return new LazyWhileLoopStatement(condition, body, ignoredCondition, locals, conditionIsTop, conditionIsUntil, _semanticModel, syntax, type, constantValue, isImplicit);
+            return new LazyWhileLoopStatement(condition, body, ignoredCondition, locals, continueLabel: null, exitLabel: null, conditionIsTop, conditionIsUntil, _semanticModel, syntax, type, constantValue, isImplicit);
```

Comment:
Why are these null instead of continue and break labels on [BoundLoopStatement](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Generated/BoundNodes.xml.Generated.cs,ed594aa427a9ad24)?

Reply:
Because I had a foolish notion that you have disabused me of.

---
In reply to: [170057138](https://github.com/dotnet/roslyn/pull/24979#discussion_r170057138) [](ancestors = 170057138)

Reply:
Added C# support.

---
In reply to: [170062363](https://github.com/dotnet/roslyn/pull/24979#discussion_r170062363) [](ancestors = 170062363,170057138)